### ISSUE TITLE
[FEATURE] Overmind VSCode expand JSON state by default

### ISSUE THREAD

**Issue by deadcoder0904** [cerebral/overmind#384]
Right now, my JSON state is always closed.

```json
{
addressBar: {error, url...}
device: {scrollPosition...}
}
```

I have to toggle it open every time to read state like this:

```json
{
  "addressBar": {
    "error": null,
    "url": "http://localhost:8080"
  },
  "device": {
    "scrollPosition": {
      "x": 0,
      "y": 0
    }
  }
}
```

I think this should be default or at least some configuration option should be provided to use with it.

PS: Overmind is a breath of fresh air. It’s so simple that anyone can understand it. Especially love the best practices in docs :)

**Comment by christianalfoni** [comment]
@deadcoder0904 Hi and thanks for the feedback!

You make a really good point here... we could automatically open on level nested objects with up to 5 keys or something. Hm... or even 10 keys. It is a bit tricky though as people structure their state very differently. But we could start experiment with this 😄

**Comment by deadcoder0904** [comment]
Maybe add a property to make it toggleable like [peacock does for favoriteColors](https://www.peacockcode.dev/guide/#favorite-colors). This will be just a `boolean`.

For me, it made sense as my state wasn't as big as yours.

**Comment by christianalfoni** [comment]
Yeah, that sounds like a good idea. Just make it on by default and as your state grows you can turn it off. Though still thinking one nested level is enough for auto-expand? Or you think every single object should be expanded?

**Comment by deadcoder0904** [comment]
I mean I want all since my application is short. Like really short (2 levels deep max) which I assume most are.

You will have better judgement on the big ones like Codesandbox :)

**Comment by christianalfoni** [comment]
Ah, I see! I was just thinking if you load like 100 items and put them into an object, it would expand all 100 items as well... which is typically not what you want 😄 

But okay, needs some configuration then, noted 😄

**Comment by deadcoder0904** [comment]
> Ah, I see! I was just thinking if you load like 100 items and put them into an object, it would expand all 100 items as well... which is typically not what you want 😄

Haha didn't think about that as it wasn't needed for my use-case. But yeah certainly don't want that.

**Comment by christianalfoni** [comment]
Okay, I think I will try to implement this as: "Keys at root level that has 5 or less keys will expand automatically". We try that first and see how it goes 😄
